spec/api/v2/market_research_library/search_spec.rb

```diff
@@ -134,4 +134,18 @@
       end
     end
   end
+
+  describe 'GET /market_research_library/search.json?size=-1' do
```

Comment:
I am not sure if we need to test the ES scroll by adding 100 documents.
If you think it is needed, please guide me on how to generate something similar to https://github.com/GovWizely/webservices/blob/master/spec/fixtures/market_research/source.txt